mcfly/models/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .model_cnn import Model_CNN
+from .model_conv_lstm import Model_ConvLSTM
+from .model_inception_time import Model_InceptionTime
+from .model_resnet import Model_ResNet
```

Comment:
Could you rename the models from 'model_CNN' to 'CNN' etc?
They are already in subpackage models.

Reply:
Good idea, it makes more sense that way

Reply:
Done.